msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgSettings.jsp

```diff
@@ -22,23 +22,23 @@
 			<h1><h:outputText value="#{msgs.pvt_msgs_label} #{msgs.pvt_settings}" /></h1>
 			<h:messages styleClass="alertMessage" id="errorMessages" rendered="#{! empty facesContext.maximumSeverity}"/>
 			
-			    <h:panelGroup styleClass="shorttext">
-					  <h:outputLabel for="" ><h:outputText value="#{msgs.pvt_actpvtmsg1}"/></h:outputLabel>
-					</h:panelGroup>
-					<h:panelGroup >
+			    <h:panelGroup styleClass="shorttext" rendered="#{PrivateMessagesTool.instructor && PrivateMessagesTool.messagesandForums}">
+					<f:verbatim><h3></f:verbatim>
```

Comment:
Im not 100% sure, but i think that the "<f:verbatim>" are not strictly necessary. 
We should try to avoid them for a cleaner code